doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -686,6 +687,12 @@ Modem libraries
     * To use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
     * The :c:func:`lte_lc_neighbor_cell_measurement` function to return an error for invalid GCI count.
 
+  * Added:
+
+    * The :c:func:`lte_lc_uiccpowersave_get` function for reading the current UICC power saving mode, and the :c:func:`lte_lc_uiccpowersave_set` function for setting the UICC power saving mode. These functions are supported by Modem Firmware 2.0.2 or higher. Please refer to the :ref:`lte_lc_readme` documentation for more information.
+    * The :c:enum:`LTE_LC_UICCPOWERSAVE_DEFAULT`, :c:enum:`LTE_LC_UICCPOWERSAVE_POLLING`, :c:enum:`LTE_LC_UICCPOWERSAVE_BYPASS`, and :c:enum:`LTE_LC_UICCPOWERSAVE_RRC` UICC power saving modes. These modes are supported by Modem Firmware 2.0.2 or higher. Please refer to the :ref:`lte_lc_readme` documentation for more information.
```

Comment:
I think this entry is not needed as it is referenced by/related to the above entry.